packages/client/src/links/wsLink.ts

```diff
@@ -283,7 +298,7 @@ export function wsLink<TRouter extends AnyRouter>(
     return ({ op }) => {
       return observable((observer) => {
         const { type, path, id, context } = op;
-
+        context.runtime = runtime;
```

Comment:
is this needed?

Reply:
That's the easiest way I found to get `runtime` into `createWSClient` without changing any of its `opts`.